site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -87,6 +87,26 @@ kubectl create -f gameserver.yaml
 kubectl apply -f gameserver.yaml
 ```
 
+{{< alert title="Note" color="info">}}
+If you changed `main.go` again and want to apply the changes to the new game servers, then you also need 
+to modify the `gamerserver.yaml` file's `imagePullPolicy` to be `Always`, or else the new game servers might
```

Comment:
It's not the game servers pulling a new image, it's `kubelet`, or just say the node.

Something like: "or the node may use a cached copy of the image, which doesn't have the new changes."